Makefile

```diff
@@ -706,9 +700,19 @@ SPEED_OPTIMISED_SRC := $(SPEED_OPTIMISED_SRC) \
             telemetry/esc_telemetry.c \
 
 SIZE_OPTIMISED_SRC := $(SIZE_OPTIMISED_SRC) \
+            drivers/adc.c \
+            drivers/dma.c \
+            drivers/pwm_output.c \
             drivers/serial_escserial.c \
+            drivers/serial_uart_stm32f30x.c \
             drivers/vtx_common.c \
+            fc/fc_init.c \
+            fc/config.c \
+            flight/servos.c \
+            flight/mixer.c \
```

Comment:
`flight/servos.c` and `flight/mixer.c` also contain functions that are part of the PID loop, so they should probably shouldn't be size optimised in their entirety.